### PR TITLE
Fixed lookups by hostname so that the ip is not returned

### PR DESCRIPTION
**Pull request by deformedonion**

### CODE REVIEW

File: src/main/java/com/oneops/inv/Inventory.java [hunk 1]
```diff
@@ -688,9 +689,9 @@ private String computeHostId(CiResource compute) throws InventoryException {
                     JSONObject entries = new JSONObject(entriesStr);
                     ArrayList<String> hostnames = new ArrayList<String>();
                     hostnames.addAll( entries.keySet() );
-                    Collections.sort( hostnames );
-
-                    hostId = hostnames.get(0);
+                    //filter out the ip returned as part of a hostname component with a nasty regex
```
Comment (mosabua) [inline src/main/java/com/oneops/inv/Inventory.java]: Wanna doc what the nasty regex is supposed to remove?

Comment (deformedonion) [reply src/main/java/com/oneops/inv/Inventory.java]: Done :)

**Comment by mosabua** [comment]
Beautiful

**Comment by mosabua** [comment]
Let me know if we need to do a release of this or so..

**Pull request merged**